crates/turborepo-api-client/src/lib.rs

```diff
@@ -123,6 +123,16 @@ pub struct APIAuth {
     pub team_slug: Option<String>,
 }
 
+impl std::fmt::Debug for APIAuth {
```

Comment:
Minor detail because `OptsInputs` derives `Debug`. This is a custom impl because printing the `token` field seemed like a bad idea

Reply:
is it possible this caused the [CI failure](https://github.com/vercel/turbo/actions/runs/9947320122/job/27480069591?pr=8759) by obscuring the token?

```
turbo run test --filter=turborepo-tests-integration --color --env-mode=strict --token=*** --team=vercel
```

Reply:
Hmm I don't think so. Lemme see what's causing that